src/components/sidebar/components/sidebar-back-to-link/style.module.css

```diff
@@ -11,6 +12,15 @@
   }
 }
 
+/*
+TODO: This is a temporary class. It should be removed when the development
+popover is removed.
+*/
+.popoverButton {
+  composes: focus-border-and-box-shadow-light from global;
+  border-radius: 5px;
+}
```

Comment:
This seems great, apologies for missing better `:focus` state in the original popover PR. Bit of a nit, but IMO we could move this into `dev-popover/style.module.css`, and remove both the `buttonClassName` and `containerClassName` from the `dev-popover` props 👍 Let me know what you think!

Reply:
I was wondering about this too, I think that'll be a good change to make. Mind if I do it in a follow-up PR to get this one merged faster?